pkg/minikube/kubeconfig/settings.go

```diff
@@ -149,3 +160,36 @@ func Update(kcs *Settings) error {
 	}
 	return nil
 }
+
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
```

Comment:
I think it would be useful to move these functions into internal_extension.go for organization

Reply:
good idea

Reply:
good idea